cli/src/web_ui.rs

```diff
@@ -22,7 +32,7 @@ macro_rules! resource {
 
 macro_rules! posix_resource {
     ($name: tt, $path: tt) => {
-        #[cfg(unix)]
+        #[cfg(all(unix, TREE_SITTER_EMBED_WASM_BINDING))]
```

Comment:
I think what we can do here is *replace* `#[cfg(unix)]` with `#[cfg(TREE_SITTER_EMBED_WASM_BINDING)]`. Basically, instead of a Posix/Windows distinction, we'll have the distinction between *embedded wasm files* and *non-embedded wasm files*.

I think we can rename the `posix_resource` macro to something like `optional_resource` - a file which *may* or *may not* be embedded in the binary.